### PR TITLE
Ignore remotewrite feature in kube-system namespace.

### PR DESCRIPTION
**Pull request by mcbenjemaa**
Ignore remotewrite feature in kube-system namespace. #1102

### CODE REVIEW

File: pkg/remotewriteutils/util.go [hunk 1]
```diff
@@ -41,5 +41,12 @@ func FetchPrometheusList(ctx context.Context, r *ResourceWrapper, rw *pmov1alpha
 		return nil, microerror.Maskf(errorFetchingPrometheus, "Could not fetch Prometheus with label selector %#q", rw.Spec.ClusterSelector.String())
 	}
 
+	for i, p := range prometheusList.Items {
```
Comment (QuentinBisson) [inline pkg/remotewriteutils/util.go:44]: Would it not be easier to change the label selector to skip the agent using the not operator and this label app.kubernetes.io/name: prometheus-agent ?

Comment (mcbenjemaa) [reply pkg/remotewriteutils/util.go:44]: it's clusterSelector and not prometheusSelector

Comment (QuentinBisson) [reply pkg/remotewriteutils/util.go:44]: Well that's how w ecall it but it's used to select prometheus is it not?

Comment (QuentinBisson) [reply pkg/remotewriteutils/util.go:44]:
```go
prometheusList, err := r.PrometheusClient.
		MonitoringV1().
		Prometheuses(metav1.NamespaceAll).
		List(ctx, metav1.ListOptions{LabelSelector: selector.String()})
```

Comment (mcbenjemaa) [reply pkg/remotewriteutils/util.go:44]: Hmmm, let me check!

Comment (QuentinBisson) [reply pkg/remotewriteutils/util.go:44]: LabelSelecto contains a MatchLabels and MatchExpressions :)

Comment (QuentinBisson) [reply pkg/remotewriteutils/util.go:44]: So we can add a MatchExpressions Not Equal app.kubernetes.io/name: prometheus-agent

Comment (mcbenjemaa) [reply pkg/remotewriteutils/util.go]: let's try

Comment (mcbenjemaa) [reply pkg/remotewriteutils/util.go]: works ✅

Comment (mcbenjemaa) [reply pkg/remotewriteutils/util.go]: can you take a look?

**Review by QuentinBisson — APPROVED** [review]
Looks way better than the loop to me :)

File: pkg/remotewriteutils/util.go (hunk 1 above)
Comment (QuentinBisson) [reply pkg/remotewriteutils/util.go]: Nice

**Pull request merged**